src/auth/authenticationManager.ts

```diff
@@ -51,3 +43,59 @@ export const useCheckMe = (path: string) => {
 
   return username;
 };
+
+export const useCheckIsLoggedIn = () => {
+  const isRefreshToken = myLocalStorage.isRefreshToken();
+  const [trigger, result] = useLazyGetIdAccessTokenQuery();
+  const [triggerLogOut, resultLogOut] = useLazyLogOutQuery();
```

Comment:
better be named `logoutResult`. The closer to plain English, the better

Reply:
renamed